sw/airborne/arch/stm32/subsystems/actuators/actuators_dualpwm_arch.c

```diff
@@ -153,13 +154,13 @@ void dual_pwm_isr(void)
   if (num_pulse == 1) {
 
     set_dual_pwm_timer_s_period(ratio_16ms);
-    set_dual_pwm_timer_s_oc(actuators_dualpwm_values[FIRST_DUAL_PWM_SERVO]);
+    set_dual_pwm_timer_s_oc(actuators_dualpwm_values[2],actuators_dualpwm_values[4]);
```

Comment:
Seems to be hardcoded now... probably not what we want...
If you really want to keep it hardcoded, at least remove the FIRST_DUAL_PWM_SERVO and SECOND_DUAL_PWM_SERVO defines...